src/main/java/edu/hm/hafner/analysis/parser/ClangAnalyzerPlistParser.java

```diff
@@ -0,0 +1,168 @@
+					.setColumnStart(extractIntField(diag, diagLocationColPath))
+                    .setCategory(extractField(diag, diagCategoryPath))
+                    .setType(extractField(diag, diagTypePath))
+                    //.setAdditionalProperties()
+                    ;
+
+                report.add(issueBuilder.build());
+            }
+
+            return report;
+        }
+        catch (XPathExpressionException e) {
+            throw new ParsingException(e);
+        }
+    }
+
+    private static XPathExpression compileDiagStrPath(final XPath xPath, final String field) throws XPathExpressionException
+    {
+        String search = "./key[text()='" + field + "']/following-sibling::string";
+        return xPath.compile(search);
+    }
+
+
+    private static XPathExpression compileDiagLocationPath(final XPath xPath, final String field) throws XPathExpressionException
+    {
+        String search = "./key[text()='location']/following-sibling::dict/key[text()='" + field + "']/following-sibling::integer";
+        return xPath.compile(search);
+    }
+
+
+    private static String extractField(final Element diag, final XPathExpression expr) throws XPathExpressionException
+    {
+        NodeList keys = (NodeList)expr.evaluate(diag, XPathConstants.NODESET);
+
+        for (Element k : XmlElementUtil.nodeListToList(keys)) {
+            return k.getTextContent();
+        }
+
+        return "";
+    }
+
+
+    private static int extractIntField(final Element diag, final XPathExpression expr) throws XPathExpressionException
+    {
+        String val = extractField(diag, expr);
+        return Integer.parseInt(val);
+    }
+
+ 
+    public static void printNode(final Node n)
+    {
+        System.out.println(n.getNodeName() + ": type (" + n.getNodeType() + "):");
+
+        short t = n.getNodeType();
+        if (t == Node.TEXT_NODE) {
+            System.out.println("!TEXT_NODE");
+            Text tx = (Text) n;
+            System.out.print("Whole text: ");
+            System.out.println(tx.getWholeText());
+        } else if (t == Node.ATTRIBUTE_NODE) {
+            System.out.println("!ATTRIBUTE_NODE");
+            System.out.println(n.getNodeValue());
+            return;
+        } else if (t == Node.ELEMENT_NODE) {
+            System.out.println("!ELEMENT_NODE");
+            Element el = (Element) n;
+            System.out.print("Tag: ");
+            System.out.println(el.getTagName());
+        }
+            
+        System.out.print("Type: ");
+        System.out.println(n.getNodeType());
+        System.out.print("Node value: ");
+        System.out.println(n.getNodeValue());
+        // Text could be long!
+        //System.out.print("Text: ");
+        //System.out.println(n.getTextContent());
+        //System.out.print("Base URI: ");
+        //System.out.println(n.getBaseURI());
+        if (n.hasChildNodes()) {
+            System.out.print("Childs count: ");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.out.print is used](https://app.codacy.com/manual/uhafner/analysis-model/pullRequest?prid=4824671)